docs/src/examples/channel.md

```diff
@@ -56,6 +55,7 @@ trait Channel {
 
    - `p`: Index of the target party for send/receive
    - `info`: Various information useful for logging
+   - `msg`: (Optional) message sent to the target party
```

Comment:
I'm not sure why the `msg` is "optional"? You mean because it's only a parameter in one of the two methods? Still, I find this phrasing quite confusing, so I would just remove the "optional" here.